policies/releasestrat.html

```diff
@@ -55,65 +56,22 @@ <h5>
 	  fixes. Before that, bug and security fixes will be applied
 	  as appropriate.</p>
 
-	  <p>The next version of OpenSSL will be 1.1.1 which will be an LTS release.
-	  This is currently in development and has a primary focus of implementing
-	  TLSv1.3. The RFC for TLSv1.3 has not yet been published by the IETF.
-	  OpenSSL 1.1.1 will not have its final release until that has happened.</p>
-
-	  <p>The draft release timetable for 1.1.1 is as follows. This may be
-          amended at any time as the need arises.</p>
-
+	  <p>The following stability rules apply to all changes made to code
+	  targeted for a version after 1.1.1:</p>
 	  <ul>
-	    <li>13th February 2018, alpha release 1 (pre1)</li>
-	    <li>27th February 2018, alpha release 2 (pre2)</li>
-	    <li>20th March 2018, beta release 1 (pre3)
-              <ul>
-	        <li>OpenSSL_1_1_1-stable created (feature freeze)</li>
-	        <li>master becomes basis for 1.1.2 or 1.2.0 (TBD)</li>
-	      </ul>
-	    <li>3rd April 2018, beta release 2 (pre4)</li>
-	    <li>17th April 2018, beta release 3 (pre5)</li>
-	    <li>1st May 2018, beta release 4 (pre6)</li>
-	    <li>29th May 2018, beta release 5 (pre7)</li>
-	    <li>19th June 2018, beta release 6 (pre8)</li>
-	    <li>Release readiness check following pre8 release (new release
-	        cycles added if required)</li>
+	    <li>No existing public interface can be removed or modified (in an
+	    API incompatible manner) until its replacement has been in place in an
+	    LTS stable release and the original function has been documented as
+	    deprecated for at least 5 years. A public interface is any function or
+	    macro declared in a public header file.</li>
```

Comment:
The language here is confusing.

If a function is modified, it means it's name remains the same but it's signature changes. 
If it's replaced, it means a new symbol is added, and the old symbol is deprecated and removed later.

Those are two different ways of doing things, and this text seems to mix them up.

Reply:
Changing types to functions, specifically `int` to `ssize_t` or `size_t` is changing an interface, but is usually a source-compatible. Or even the length of the options field/API from `long` to `uin64_t`. These types of changes ("size_t'ifying") should be allowed at ABI-breaking boundaries without these restrictions.